src/migration/migrationRules.js

```diff
@@ -486,4 +486,13 @@ module.exports = {
       }
     })
   },
+  25: (config) => {
+    config.engine.caching.archive = {
+      enabled: config.engine.caching.archiveMode === true,
+      archiveFolder: config.engine.caching.archive.archiveFolder,
+      retentionDuration: 8760,
```

Comment:
That means that someone who update OIBus with an activated archive mode will lose its data older than one year. It is a breaking change compared to the old archive management (where everything is kept forever). 
We may add another behaviour : after the migration, the new value will be 0 (instead of one year). The input in the front should accept a minimum value of 0 and tell in the help that 0 stands for 'infinite retention period'.
In the backend, you must check if the rettentionDuration is greater than 0 before activating a deletion cycle.